ecocode-rules-specifications/src/main/rules/EC36/html/EC36.asciidoc

```diff
@@ -11,25 +11,33 @@ However, even without autoplay, segments of video or audio files might still dow
 This leads to unnecessary data usage, especially if users don't commence playback.
 To mitigate this, it's crucial to prevent browsers from preloading any content by configuring the appropriate settings.
 
-Video:
+== How to fix it in video
```

Comment:
I think both section names are wrong here, we do not provide a video to explain how to fix the problem
```suggestion
== How to fix a video element
```

Reply:
Those section titles were changed in latests commits.